Lib/test/test_ctypes/test_wintypes.py

```diff
@@ -38,6 +41,22 @@ def test_variant_bool(self):
         vb.value = []
         self.assertIs(vb.value, False)
 
+    def test_signess(self):
+        # Unsigned aliases with octet sie in their name
+        for type in (wintypes.BYTE, wintypes.WORD, wintypes.DWORD):
+            with self.subTest(type=type):
+                self.assertEqual(type(200).value, 128)
```

Comment:
Does this pass?

```python
>>> ctypes.wintypes.WORD(200).value
200
```

Reply:
Let's make signedness test a WinTypesTest method and implement it like this:

```python
    def assertIsSigned(self, ctype):
        self.assertLess(ctype(-1).value, 0)
```

Reply:
Thanks for the advice. I also got rid of the second loop over unsigned types to avoid further confusion between similar looking code blocks.